src/decision/test/lidar-decision-test.cpp

```diff
@@ -7,6 +7,8 @@
 #include <LidarDecision.h>
 #include <gtest/gtest.h>
 
+#define distance 1.0f
```

Comment:
I simply used `1.0` directly similar to the angle parameters which are written as `0.02`.